controlplane/kubeadm/controllers/controller_test.go

```diff
@@ -48,24 +43,78 @@ import (
 	"sigs.k8s.io/cluster-api/util"
 	"sigs.k8s.io/cluster-api/util/kubeconfig"
 	"sigs.k8s.io/cluster-api/util/secret"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+var _ = Describe("KubeadmControlPlaneReconciler", func() {
+	BeforeEach(func() {})
+	AfterEach(func() {})
```

Comment:
Are we creating these placeholders to inform others that these hooks exist?

Reply:
yeah, no need.